lib/userRequestHelpers.js

```diff
@@ -7,11 +7,16 @@ export const retrievePublic = (uid) => {
     return axios.get(baseUrl + `/api/u/users/${uid}`)
 };
 export const updatePersonal = (uid, token, field, update) => {
+    console.dir('IDIDIDID:::' + uid + 'AUTHAUTH:' + token)
```

Comment:
At some point, delete this console log, but yeah getting auth has been hard

Reply:
Haha yeah still didn't get there :(